### PR TITLE
Cu 86bzf9c4p update platform je mpi images

### PR DESCRIPTION
**Pull request by MatthewErispe**
## Summary by CodeRabbit

- **New Features**
	- Introduced structured schemas for managing patient and auxiliary demographic data fields, enhancing data integrity and user experience.
	- Added new services in Docker configurations, allowing targeted deployment for improved resource management.
	- Enhanced Dgraph cluster setup with additional services and improved scalability.

- **Bug Fixes**
	- Increased file upload limit to 128MB in secure HTTP configuration for better handling of data submissions.
	- Updated default password for OpenHIM API to strengthen security.

- **Chores**
	- Updated metadata and configuration settings, improving overall application functionality and maintainability.

### CODE REVIEW

**Comment by rcrichton** [comment]
Task linked: [CU-86bzf9c4p Update Platform JeMPI images](https://app.clickup.com/t/86bzf9c4p)

**Review by rcrichton — COMMENTED** [review]
Looks good to me. Just a few comments.

File: client-registry-jempi/docker-compose.api-cluster.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+  jempi-api-kc:
+    deploy:
+      placement:
+        constraints:
+          - "node.labels.name==node-2"
```
[the remark this answers is not in the document]
Comment (MatthewErispe) [reply client-registry-jempi/docker-compose.api-cluster.yml:8]: @rcrichton I think I added these when testing. I can revert this changes

File: client-registry-jempi/docker-compose.combined-cluster.yml [hunk 2]
```diff
@@ -6,3 +6,15 @@ services:
...
+    deploy:
+      placement:
+        constraints:
+          - "node.labels.name==node-1"
+  
+  jempi-async-receiver:
+    deploy:
+      placement:
+        constraints:
+          - "node.labels.name==node-2"
```
[the remark this answers is not in the document]
Comment (MatthewErispe) [reply client-registry-jempi/docker-compose.combined-cluster.yml:20]: @rcrichton I added this when testing. This can be reverted

**Review by rcrichton — APPROVED** [review]
LGTM

**Pull request merged**